src/olympia/addons/management/commands/process_addons.py

```diff
@@ -30,6 +31,16 @@
     'addon_review_aggregates': {'method': addon_review_aggregates, 'qs': []},
     'sign_addons': {'method': sign_addons, 'qs': []},
     'unsign_addons': {'method': unsign_addons, 'qs': []},
+    'update_latest_version': {
+        'method': update_latest_version,
+        'qs': [Q(type__in=[amo.ADDON_EXTENSION,
+                           amo.ADDON_THEME,
+                           amo.ADDON_DICT,
+                           amo.ADDON_LPADDON,
+                           amo.ADDON_PLUGIN,
+                           amo.ADDON_API]),
```

Comment:
I wanted to exclude `amo.ADDON_PERSONA` because the `latest_version` handling is a little different for them.
I'll remove `ADDON_API` though.